fedora/__init__.py

```diff
@@ -373,3 +377,13 @@ async def whoowns(self, evt: MessageEvent, package: str) -> None:
         resp = "".join([x for x in [owners, admins, committers] if x != ""])
 
         await evt.reply(resp)
+
+    @command.new(help="Retrive a user's pronouns")
+    @command.argument("username", pass_raw=True, required=False)
+    async def pronouns(self, evt: MessageEvent, username: str) -> None:
```

Comment:
hmmm, not sure we need a seperate command for this -- is returning them with the userinfo not enough?

Reply:
I like having a simple, one line output command for this, and I don't think it adds too much of a maintenance version

Reply:
i just worry about command bloat really -- dont want to be adding commands for each field in a user's Fedora Account.